src/wasm-interpreter.h

```diff
@@ -35,6 +35,24 @@ using namespace cashew;
 
 IString WASM("wasm");
 
+
+#ifdef WIN32
+#include <intrin.h>
+
+int32_t safe_clz(int32_t v) {
+  unsigned long result;
+  return _BitScanReverse(&result,v) ? result : 32;
+}
+
+int32_t safe_ctz(int32_t v) {
+  unsigned long result;
+  return _BitScanForward(&result,v) ? result : 32;
+}
+
+int32_t platform_popcount(int32_t v) {
+  return __popcnt(v);
```

Comment:
You're right, this doesn't work on CPUs without the `popcnt` instruction. I think this change is still an improvement on not compiling, though. :)